tools/lib.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Copyright (c) 2023 Petr Vorel <pvorel@suse.cz>
+
+ask()
+{
+	local msg="$1"
+	local answer
+
+	printf "\n$msg. Proceed? [N/y]: "
+	read answer
+	case "$answer" in
+		[Yy]*) : ;;
+		*) exit 2
+	esac
+}
+
+quit()
+{
+	printf "\n$@\n" >&2
+	exit 1
+}
+
+rod()
+{
+	eval "$@" || quit "$@ failed"
```

Comment:
yikes, don't use `eval`.  it's not even necessary here -- just delete it.

also, use `$*`, not `$@` when calling `quit`

Reply:
> yikes, don't use eval. it's not even necessary here -- just delete it.

This is inspired by LTP, which uses `tst_rod` binary. I'm not sure if everything would work without `eval`, but maybe you're right.